src/js/components/Diagram/stories/Animated.js

```diff
@@ -106,7 +90,7 @@ export const Animated = () => {
                 ))}
               </Box>
             </Box>
-            <Diagram connections={connections} />
+            <Diagram animation={{ duration: 3000 }} connections={connections} />
```

Comment:
For the example we might want to demonstrate an example that is more detailed and add in the `type`

Reply:
Totally agree, added the `type` prop back in